Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
```

Comment:
The commit message header is incorrect. It should be `[release-4.13] Update version to 8.0.2`